modules/systemsmanager__rce_ec2/main.py

```diff
@@ -155,7 +155,7 @@ def main(args, pacu_main):
         )['Policy']
 
         if ssm_policy['AttachmentCount'] > 0:
-            if fetch_data(['IAM', 'Roles'], 'enum_users_roles_policies_groups', '--roles') is False:
+            if fetch_data(['IAM', 'Roles'], 'iam__enum_users_roles_policies_groups', '--roles') is False:
```

Comment:
Add `iam__enum_users_roles_policies_groups` to prereq modules and remove the hardcode here